backend/build.gradle

```diff
@@ -105,10 +108,6 @@ dependencies {
     testCompile "com.github.tomakehurst:wiremock-jre8:2.28.1"
 
     // Forcing these version resolutions over the BOM of spring-cloud-dependencies below
```

Comment:
Can this comment on line 110 be moved to follow the relocated lines, or purged if no longer needed?

Reply:
good point, might be misleading